code_generator/GdkCodeGenerator/Templates/UnityComponentConversionGenerator.tt

```diff
@@ -69,6 +69,23 @@ namespace <#= qualifiedNamespace #>
                 entityManager.AddComponentData(entity, data);
                 entityManager.AddComponentData(entity, new NotAuthoritative<<#= componentDetails.TypeName #>>());
 
+                var updatesComponent = new <#= componentDetails.TypeName #>.ReceivedUpdates
+                {
+                    handle = ReferenceTypeProviders.UpdatesProvider.Allocate(World)
+                };
+                
+                ReferenceTypeProviders.UpdatesProvider.Set(updatesComponent.handle, new List<<#= componentDetails.TypeName #>.Update>());
+                entityManager.AddComponentData(entity, updatesComponent);
+
+                var update = new <#= componentDetails.TypeName #>.Update 
+                {
+<# foreach (var fieldDetails in fieldDetailsList) { #>
+                    <#= fieldDetails.PascalCaseName #> = data.<#= fieldDetails.PascalCaseName #>,
+<# } #>
+                };
```

Comment:
Would prefer the following order:

```
1. Create the update
2. Create the list (can initialise with the update
3. Create the component and allocate,
4. Set the list
5. Add component to entity.
```

Also you are missing the `entityManager.AddComponentData` 😉

Reply:
Reordered and added codegen.